datafusion/core/tests/sqllogictests/test_files/interval.slt

```diff
@@ -190,25 +190,25 @@ select interval '1 year' - '1 month' - '1 day' - '1 hour' - '1 minute' - '1 seco
 query ?
 select -interval '5' - '1' - '2' year;
 ----
-0 years -24 mons 0 days 0 hours 0 mins 0.000000000 secs
+0 years -96 mons 0 days 0 hours 0 mins 0.000000000 secs
```

Comment:
To me this doesn't look right. The previous test seems correct. When putting a negation in front of interval I believe it should negate the entire interval statement.

`interval '5' - '1' - '2' year` is one interval statement.

I think the statement

```
select -interval '5' - '1' - '2' year;
```

Should be interpreted as:

```
select - (interval '5' - '1' - '2' year);
```

Rather than

```
select (- interval '5') - '1' - '2' year;
```

If the user wants to negate only `5`, they should write it as follows:

```
select interval - '5' - '1' - '2' year;
```

Or:

```
select - interval '5' year - interval '1' year - interval '2' year;
```

Anyway, just my opinion.

Reply:
cc @jackwener

Reply:
Agree with you, I think we should notice this point. cc @alamb .
If we will merge this PR, I think we need to add a ticket to follow this problem

Reply:
I will wait for @waitingkuo 's comments -- I agree about filing the ticket if we merge